src/Workspaces/CSharp/Portable/Extensions/ContextQuery/SyntaxTreeExtensions.cs

```diff
@@ -1201,7 +1201,7 @@ public static bool IsParameterTypeContext(this SyntaxTree syntaxTree, int positi
 
             if (token.IsKind(SyntaxKind.OutKeyword) &&
                 token.Parent.IsKind(SyntaxKind.Argument) &&
-                ((ArgumentSyntax)token.Parent).Declaration != null)
```

Comment:
"strong" is a matter of perspective :)

I don't like the check because it doesn't appear to serve any purpose.  I find it more confusing because i have to ask the questin to myself "when do i have an 'out' keyword parented by an Argument that *isn't* the refOrOutKeyword of the argument".  Since hte answer is "never", i'd prefer to not bother with the check.